src/components/ColorRampDropdown.tsx

```diff
@@ -41,20 +73,20 @@ const ColorRampSelector: React.FC<ColorRampSelectorProps> = (propsInput): ReactE
     const colorRampEntries = Array.from(props.colorRamps!.entries());
     // Make a button for every color ramp
     for (let i = 0; i < props.colorRamps.size; i++) {
-      let className = "";
       // Manipulate class names for rounding at start and end of dropdown list
+      let className = "";
       if (i === 0) {
-        className += " " + styles.dropdownFirst;
+        className = styles.dropdownFirst;
       }
       if (i === props.colorRamps!.size - 1) {
-        className += " " + styles.dropdownLast;
+        className = styles.dropdownLast;
```

Comment:
tip: this can also be accomplished with the CSS `:first-child` and `:last-child` pseudo-classes